src/pages/api/parts.ts

```diff
@@ -14,6 +20,22 @@ import {
 } from "../../lib/api";
 import { getClient, makeCall } from "../../lib/vertex-api";
 
+export interface GetPartsRes extends Res {
+  readonly cursor?: string;
+  readonly data: PartData[];
+}
+
+export interface CreatePartBody {
```

Comment:
Instead of doing this, I've started using the `Pick` Typescript feature so these stay up-to-date as the API changes. See https://github.com/Vertexvis/vertex-dev-dashboard/blob/ad58ab889ce52811d78f4e11a054c6c480e74dac/src/pages/api/scenes.ts#L24-L25